app/imports/api/users/server/publications.js

```diff
@@ -0,0 +1,13 @@
+import { Meteor } from 'meteor/meteor';
+
+Meteor.publish('userList', () => {
+    const query = {};
+
+    const options = {
+        fields: {
+            services: 1,
```

Comment:
Publishing user access tokens, mails and other private data is never good choice!